src/main/java/com/google/devtools/build/lib/profiler/JsonTraceFileWriter.java

```diff
@@ -172,13 +172,16 @@ public void run() {
               // The buffer size of 262144 is chosen at random.
               new OutputStreamWriter(
                   new BufferedOutputStream(outStream, 262144), StandardCharsets.UTF_8))) {
+        var finishDate = new Date();
         writer.beginObject();
         writer.name("otherData");
         writer.beginObject();
         writer.name("bazel_version").value(BlazeVersionInfo.instance().getReleaseName());
         writer.name("build_id").value(buildID.toString());
         writer.name("output_base").value(outputBase);
-        writer.name("date").value(new Date().toString());
+        // deprecated, use profile_finish_ts instead
```

Comment:
Let's not deprecate the field, it is much easier for humans to parse than a timestamp :-)

Reply:
I don't like how ambiguous the field name is though. It misled me to think it was build start time but in reality, it's build finished time.

Do you think we could rename it?

Reply:
We could, but let's do it separately. This change can be cherry-picked then into a minor version and we only change the field name in the next major version.